pom.xml

```diff
@@ -148,7 +148,7 @@
       <dependency>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>animal-sniffer-annotations</artifactId>
-        <version>1.14</version>
+        <version>1.16</version>
```

Comment:
[Changes](https://github.com/mojohaus/animal-sniffer/compare/animal-sniffer-parent-1.14...animal-sniffer-parent-1.16). I suppose the one you hit was https://github.com/mojohaus/animal-sniffer/pull/32.